README.md

```diff
@@ -23,38 +25,23 @@ It's common for developers to create a website to show off their web-dev skills
 - Practice applying visual styles with CSS
 - Use both HTML & CSS together to create a comprehensive design
 
-## Objective
-
-Use HTML & CSS to create a static site whose content is your portfolio. Your personal portfolio site should contain information about you and the coding work you've done.
-
 ## Project Requirements
 
 ### Only Use Static HTML and CSS
 
-You are only permitted to use static HTML and CSS for this project. Preprocessors (haml, erb, sass, less, etc.) and Javascript of any kind are not allowed. All submitted HTML needs to pass as valid HTML through an HTML Validator.
+We should only use static HTML and CSS for this project. Preprocessors (haml, erb, sass, less, etc.) and Javascript of any kind are not allowed. All submitted HTML needs to pass as valid HTML through an HTML Validator.
```

Comment:
```suggestion
We should only use static HTML and CSS for this project. Preprocessors (haml, erb, sass, less, etc.) and Javascript of any kind are not allowed. All submitted HTML needs to pass as valid HTML through an [HTML Validator](https://validator.w3.org/).
```

might as well make it as easy as possible for them to remember to validate their html